wallet-new/README.md

```diff
@@ -1,26 +1,49 @@
 # Cardano Wallet API
 
-May this be the beginning of a beautiful program.
+This is the Wallet Backend for a Cardano node.
 
 ## Installation
 
 Installation follows the standard approach to installing Stack-based projects.
 
 1. Install the [Haskell `stack` tool](http://docs.haskellstack.org/en/stable/README).
-2. Run `stack install` to install this package.
+2. Run `stack install cardano-sl-wallet-new` from the project *root* to install this package.
 
-## Creating a Client
+## The API
 
-TBD.
+We describe how to interact with our API via the popular [Swagger](https://swagger.io/) framework & format.
+In order to do so, we export the full Swagger specification inside `spec/swagger.json`. Such JSON file must
+be kept in sync with the current version of the project and is developer responsibility to do so upon
+committing new work.
+This will be made automatic as part of [this issue](https://iohk.myjetbrains.com/youtrack/issue/CSL-1939) but
+for now requires self-enforced discipline.
 
-## Creating a Server
+Currently the only way to generate an updated `swagger.json` is to run the `wallet-new-server` node, so that
+the updated Swagger file will be written on disk. For example:
 
-TBD.
+```
+stack exec wallet-new-server -- --topology=wallet-new/topology-examples/testnet.yaml \
+  --configuration-key mainnet_staging_short_epoch_full --wallet-debug --rebuild-db
+```
 
-## Swagger API
+Running the command above *from the root of the Cardano project* will store an updated `swagger.json` into
+`wallet-new/spec`.
 
-We export the full Swagger specification inside `spec/swagger.json`.
+### Playing with the API
+
+Once you have your updated `swagger.json` file, the easiest way is to head over to [the online editor](https://editor.swagger.io),
+click "Edit -> Import File" and import the file. Once done that, the API will be rendered in its full glory.
+
+Alternatively (and *recommended*), is also possible to download the editor locally and play with it. This has the advantage
+you can now also *try out* the API, because the `host` of the swagger API points to `localhost`, which won't work, of course,
+in case of the online editor. We won't get too deep into how to setup the editor locally, but generally speaking it should be
+as simple as:
+
+- Downloading [the editor](https://github.com/swagger-api/swagger-editor/archive/v3.1.17.zip) online;
```

Comment:
By the way, this editor can be launched simply by browser, e.g. `firefox swagger-editor/index.html`. Perhaps it's worth mentioning

Reply:
true, that's useful